modules/Material/Toolbar.qml

```diff
@@ -153,7 +153,7 @@ View {
 
     Row {
         id: windowControls
-
+        visible: clientSideDecorations
```

Comment:
Yes, a new commit would ideally be best, so in the unlikely event we could easily revert a speicific commit without reverting unrelated behavior.